### ISSUE TITLE
Support more DB's

### ISSUE THREAD

**Issue by Jogai** [aunefyren/poenskelisten#9]
Its in the FAQ, but regardless; Maybe support sqlite &&/|| postgres?

**Comment by aunefyren** [comment]
Agreed, just need to test with Gorm. Should be available in v2.0.0.

**Comment by grasdk** [comment]
SQLite on a volume in the docker container would be awesome for smaller setups. @aunefyren : You mentioned "testing with Gorm", does that mean that support is near?

Thanks for sharing your work by the way :-)

**Comment by aunefyren** [comment]
`v2.2.0` supports Postgres! I will keep this issue open to montor if SQLite support is wanted.

**Comment by aunefyren** [comment]
SQLite was implemented in `v2.3.0`. Thank you for the engagement!
